src/app.js

```diff
@@ -31,7 +31,14 @@ class Messenger extends EventEmitter {
   /*:: greetings: RegExp */
   /*:: help: RegExp */
   /*:: options: Object */
-  constructor({hookPath = '/webhook', linkPath = '/link', emitGreetings = true, cache} = {}) {
+  /*:: pages: Object */
+  constructor({
+      hookPath = '/webhook',
+      linkPath = '/link',
+      emitGreetings = true,
+      cache,
+      pages = {}
```

Comment:
Sanity check: reading through this, the expectation for the `pages` object is a key-value store where the page ID is the key and the page access token is the value. Is that correct? This appears to be validated with what is in the _possible risks or adverse effects_ section of the summary.